pyrene/src/components/Modal/Modal.examples.tsx

```diff
@@ -1,6 +1,22 @@
 import React from 'react';
 import Placeholder from '../../examples/Placeholder';
 
+
+const findTriggerButton = () => {
+  const buttons = document.getElementsByTagName('button');
+  const searchText = 'Trigger Modal';
+  const buttonArray = Array.from(buttons);
```

Comment:
usually you do plural a variable as convention for developers to know it s about a collection. That way :

```
const buttons = Array.from(buttons);
```

Reply:
I would suggest something like 
`const buttonsArray` 
because we already have a 
`const buttons` 
unless we do 
`const buttons = Array.from(document.getElementsByTagName('button'));`